content/docs/components/serving/bentoml.md

```diff
@@ -0,0 +1,234 @@
+```
+
+Use BentoML CLI to start a local API model server:
+
+```shell
+bentoml serve IrisClassifier:latest
+```
+
+BentoML automatically processes the incoming data into the required data format defined in
+the API. For the iris classifier BentoService defined above, incoming data transforms
+into a `pandas.DataFrame`.
+
+Use `curl` request in another terminal to get the prediction result:
+
+```bash
+curl -i \
+  --header "Content-Type: application/json" \
+  --request POST \
+  --data '[[5.1, 3.5, 1.4, 0.2]]' \
+  localhost:5000/predict
+```
+
+Find the file directory of the SavedBundle with `bentoml get` command, which is
+directory structured as a docker build context. Running docker build with this
+directory produces a docker image containing the model API server. Replace
+`{docker_username}` with your Docker Hub username and run the following code:
+
+```shell
+saved_path=$(bentoml get IrisClassifier:latest -q | jq -r ".uri.uri")
+
+docker build -t {docker_username}/iris-classifier $saved_path
+docker push {docker_username}/iris-classifier
+```
+
+### Deploy model server to Kubernetes
+
+The following is an example YAML file for specifying the resources required to run and
+expose a BentoML model server in a Kubernetes cluster. Replacing `{docker_username}`
+with your Docker Hub username and save it to `iris-classifier.yaml`:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: iris-classifier
+  name: iris-classifier
+  namespace: kubeflow
+spec:
+  ports:
+  - name: predict
+    port: 5000
+    targetPort: 5000
+  selector:
+    app: iris-classifier
+  type: LoadBalancer
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: iris-classifier
+  name: iris-classifier
+  namespace: kubeflow
+spec:
+  selector:
+    matchLabels:
+      app: iris-classifier
+  template:
+    metadata:
+      labels:
+        app: iris-classifier
+    spec:
+      containers:
+      - image: {docker_username}/iris-classifier
+        imagePullPolicy: IfNotPresent
+        name: iris-classifier
+        ports:
+        - containerPort: 5000
+```
+
+Use `kubectl` CLI to deploy the model API server to the cluster
+
+```shell
+kubectl apply -f iris-classifier.yaml
+```
+
+### Send prediction request
+
+Use `kubectl describe` command to get the `NODE_PORT`
+
+```shell
+kubectl describe svc iris-classifier --namespace kubeflow
+```
+
+And then send the request:
+
+```shell
+curl -i \
+  --header "Content-Type: application/json" \
+  --request POST \
+  --data '[[5.1, 3.5, 1.4, 0.2]]' \
+  http://EXTERNAL_IP:NODE_PORT/predict
+```
+
+### Monitor metrics with Prometheus
+
+#### Perquisites
+
+- Prometheus installed in the cluster
+  - [Prometheus documentation](https://prometheus.io/docs/introduction/overview/)
+  - [Installation instruction with Helm chart](https://github.com/helm/charts/tree/master/stable/prometheus)
+
+BentoML provides Prometheus support out of the box for the model API server, includes
```

Comment:
BentoML API server provides Prometheus support out of the box. It comes with a "/metrics" endpoint which includes ....